### PR TITLE
CAW-479: Add javascript and css resources when the views are loaded.

### PR DESCRIPTION
**Pull request by sherakama**
Added hooks for adding Megans css to the faq pages. I have added the preprocess page function on top of the pre_render function as often we clone the views to a different name before making changes.

### CODE REVIEW

**Comment by sherakama** [comment]
@jbickar, Please review.

File: stanford_faq.module [hunk 1]
```diff
@@ -7,11 +7,34 @@
 include_once 'stanford_faq.features.inc';
 
 /**
+ * Implements hook_preprocess_HOOK().
+ */
+function stanford_faq_preprocess_page(&$vars) {
+  $current_path = current_path();
```
Comment (jbickar) [inline stanford_faq.module]: Why is this needed? arg(0) should work regardless of calling current_path(), no?

**Comment by jbickar** [comment]
Looks good other than the one question about current_path().

**Comment by sherakama** [comment]
Thanks!
